### ISSUE TITLE
Connessione verso https self-manged con let's encrypt

### ISSUE THREAD

**Issue by giuseppelanzi** [progamma/cloud-connector#30]
Collegandosi ad un server di produzione self-managed con certificato https di let's encrypt (con altri non abbiamo provato) si ottiene "transport error: 503" nell'apertura della socket.

Per aggirare il caso è stato necessario impostare _opt.rejectUnauthorized = false_ nella Node.Server.prototype.connect.

Può valere la pena metterlo dentro come parametro di configurazione o simili? Oppure ce ne dimentichiamo perché il problema si è posto una volta sola?

@andreamaioli @matteosirri che ne pensate?

**Comment by lbaldini** [comment]
Qui, però, dicono che sarebbe meglio non farlo perché il server è attaccabile dal ManInTheMiddle:

https://stackoverflow.com/questions/31861109/tls-what-exactly-does-rejectunauthorized-mean-for-me

**Comment by giuseppelanzi** [comment]
Vero. Se lo mettiamo come parametro poi basterà cambiare la configurazione e poi aggiornare il cloud-connector non causerà problemi. Altrimenti basta ricordarsi di aggiungere la riga (per quelli a cui serve) nel posto giusto. Nel caso lo faremo via supporto.
Potrebbe essere interessante sapere che è successo, chiudere questa issue ed eventualmente riproporla se ricapita altre 2 volte?
